### ISSUE TITLE
Wiki updates: Consumables

### ISSUE THREAD

**Issue by NitramLegov** [comakingspace/do-something#26]
I think it would be great if each Tool page contains a section about consumables - and whether they need to be bought be everybody themselves or are provided by the space with a usage-based donation system.

**Comment by Pakue95** [comment]
I agree. Right now I'm trying to get a sensible selection for screws and sanding materials which will be kept in stock. All the 'non-standard' stuff will get its own (far away :] ) corner and won't be restocked. 

Afterward, we can make a list for all the other tools and their standard consumables, like disc grinder, reciprocating/jig saw etc. However I'd like to limit it to around 2-3 uniques per machine and from the same manufacturer where possible (e.g. drill won't obviously work) to reduct the number of items to keep track of.

I'd like to make this initial (re-)stock all/most tool materials in one go and put it up as one "next to buy top 5" as we discussed during the most recent members plenum.

**Comment by o-liver** [comment]
Is this still an issue? AFAIK there is no consumables section yet. As a new member I don't know what consumables exist and what needs to be bought. But I think I could introduce a section to the Wiki, without filling said section. Is there any machine with such a section already in the Wiki, which could be used as a template?

**Comment by Pakue95** [comment]
For screws there are these two wiki pages:

https://wiki.comakingspace.de/Available_Wood_Screws
https://wiki.comakingspace.de/Available_Metric_Screws

Some consumables are listed on various other pages: https://wiki.comakingspace.de/index.php?title=Special%3ASearch&search=consumables&go=Go

If you have suggestions/ideas to make it more user friendly feel free to implement them :)

**Comment by NitramLegov** [comment]
I don´t think there will be not much activity on this one anymore. Therefore, I am closing this
